src/settings.js

```diff
@@ -1464,6 +1464,14 @@ var UBSAN_RUNTIME = 0;
 // by -fsanitize=leak instead of used directly.
 var USE_LSAN = 0;
 
+// Whether we should link in ASan's runtime library. This is intended to be used invoked`
```

Comment:
"used invoked" looks like a typo. also there seems to be a stay backtick at the end there